src/Parser/BalanceDruid/Modules/Spells/NewMoon.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import SpellIcon from 'common/SpellIcon';
+import SpellLink from 'common/SpellLink';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+import { formatPercentage } from 'common/format';
+import Module from 'Parser/Core/Module';
+import SPELLS from 'common/SPELLS';
+import isMoonMoon from '../Core/isMoonMoon';
+
+class NewMoon extends Module {
+
+    newMoonOrder = 0;
+    firstMoonTime = 0;
+    firstCastTime = 0;
+    firstCast = false;
+    firstMoonCast = false;
+    orderFound = false;
+
+    on_byPlayer_cast(event) {
+        if (!this.firstCast) {
+            this.firstCastTime = event.timestamp;
+            this.firstCast = true;
+        }
+
+        const spellId = event.ability.guid;
+        if (this.orderFound || !isMoonMoon(event)) {
+            return;
+        }
+
+        if (!this.firstMoonCast){
+            this.firstMoonTime = (event.timestamp - this.firstCastTime) / 1000;
+            this.firstMoonCast = true;
+        }
+
+        if (spellId !== SPELLS.NEW_MOON.id)
+            this.newMoonOrder ++;
+        else
+            this.orderFound = true;
+    }
+
+    suggestions(when) {
+        const abilityTracker = this.owner.modules.abilityTracker;
+        
+        const offSet = this.firstMoonTime + 15;
+        const totalFromCD = ((this.owner.fightDuration/1000) - offSet) / 15;
+        const eachMoon = Math.floor(totalFromCD / 3);
+        let nmAvailableCasts = eachMoon + 1;
+
+        const extraMoons = ((totalFromCD / 3) - eachMoon) * 3;
+        if (extraMoons > this.newMoonOrder) nmAvailableCasts ++;
+
+        const nmCasted = abilityTracker.getAbility(SPELLS.NEW_MOON.id).casts;
+        const percCasted = nmCasted / nmAvailableCasts;
+
+        when(percCasted).isLessThan(1)
+        .addSuggestion((suggest, actual, recommended) => {
+            return suggest(<span> Your <SpellLink id={SPELLS.NEW_MOON.id} /> cast efficiency can be improved, try maintaining yourself at low Moon charges at all times; you should (almost) never be at max (3) charges.</span>)
```

Comment:
`try maintaining yourself at low Moon charges at all times`

I had to read this a few times before I understood it, could you please change it a bit to make it clearer?

Reply:
English not my main language :D
Is this enough? If not, can you provide an alternative, please? ^^u

Reply:
I suggested `Your New Moon cast efficiency can be improved. Try keeping at least 1 charge on cooldown; you should (almost) never be at max charges.` earlier in Discord. ;)